docs/source/get_started/custom_ui.rst

```diff
@@ -400,6 +400,12 @@ Those variables let you decide which buttons will be visible in the embedded MDS
 
 .. code-block:: javascript
 
+    $scope.showToolbox
```

Comment:
something wrong here, the description about showToolbox should be above this variable